test/helm/rbac_drift_test.go

```
@@ -124,8 +238,12 @@ func helmManagerRules(t *testing.T, chartDir string) []rbacv1.PolicyRule {
 	// scoped to "only granted when feature X is enabled" is fine in
 	// principle, but it has to be tied to a runtime signal the
 	// operator code itself observes — not a Helm value the operator
-	// binary knows nothing about.
-	cmd := exec.Command("helm", "template", "drift-test", chartDir)
+	// binary knows nothing about. csiDriver.enabled is the one toggle
+	// that qualifies: it is what sets ENABLE_CSI_DRIVER, so the
+	// controller that needs the gated rules only runs when they are
+	// granted. TestHelmManagerRoleCSIGate holds the two together.
+	args := append([]string{"template", "drift-test", chartDir}, extraArgs...)
+	cmd := exec.Command("helm", args...)
```

golangci-lint (2.13.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)